src/Sylius/Bundle/CoreBundle/Resources/config/app/messenger.yaml

```diff
@@ -28,6 +28,7 @@ framework:
             'Sylius\Bundle\CoreBundle\CatalogPromotion\Command\DisableCatalogPromotion': main
             'Sylius\Bundle\CoreBundle\CatalogPromotion\Command\RemoveCatalogPromotion': catalog_promotion_removal
             'Sylius\Bundle\CoreBundle\CatalogPromotion\Command\RemoveInactiveCatalogPromotion': catalog_promotion_removal
+            'Sylius\Bundle\CoreBundle\PriceHistory\Message\ApplyLowestPriceOnChannelPricings': main
```

Comment:
As you partially see even in this file, in the CatalogPromotion concept and also in ApiBundle, we are using Command, CommandHandler and CommandDispather names instead of the ones with the `Message` prefix. I know it is not fully consistent across the entire Sylius, but I would be in favour of using Command also in this newly added feature 🖖🏻

Reply:
Reworked ✋